src/redprl/refiner_types.fun

```diff
@@ -799,7 +799,7 @@ struct
         val goal1 = makeMem (I, H) (m, (dfun, l, k))
         val goal2 = makeEqIfDifferent (I, H) ((m', n), (dfun, NONE, K.top))
       in
-        |>: goal1 >:? goal2 #> (I, H, trivial)
+        |>:? goal2 >: goal1 #> (I, H, trivial)
```

Comment:
Out of curiosity, how come you prefer this order?

Reply:
I feel in my experiments `goal2` is the "real" goal. Image you are proving `p = p'` in some path type. The goal with an eta-expanded term is the "real" one, and `goal1` is like some sanity checking.

Reply:
ok! Cool.